js/notespage.ts

```diff
@@ -8,8 +8,6 @@ let newNote: HTMLInputElement = document.querySelector("#new-note") as HTMLInput
 let newNoteDescription = document.querySelector("#new-note-description") as HTMLTextAreaElement
 let newDate: HTMLInputElement = document.querySelector('#new-date') as HTMLInputElement;
 let notesForm: HTMLFormElement = document.querySelector("#note-form") as HTMLFormElement;
-const addNoteSubmitButton = document.querySelector("#add-note") as HTMLButtonElement;
```

Comment:
add to notespage.ts (required for event listener on modal for add new todo note)